GoogleForCreators/web-stories-wp
# Shopping: Products JSON Schema Validation

## Context


In support of #11210 we need make the WoocCommerce plugin available with some sample products to query against to ensure a valid JSON schema is returned.

> Using this JSON schema for shoppable products, we can verify that the data we return from WooCommerce & Shopify matches the expected format. So most likely this is something for the PHPUnit integration tests.




## Summary


- Adds  WoocCommerce  plugin installation step to start script
- Adds sample product import



## Relevant Technical Choices



## To-do
- e2e test is failing which is expected as the value for product price isn't matching the schema



## User-facing changes



## Testing Instructions

- `npm run env:start`
-  note CLI outputs that WooCommerce is installed
-  note in the WP Admin - WooCommerce has imported sample products (via the sample xml file)






This PR can be tested by following these steps:

1. Run `npm run env:start` --- this should install WooCommerce + sample products
2. Activate the WooCommerce plugin --- ensure sample products exists


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Partially addresses #11210
